### ISSUE TITLE
How to chagne colour of external link (unreadable)

### ISSUE THREAD

**Issue by hchaase** [OokTech/TW5-Bookmarks#2]
Hi
I enjoy using your old bookmark plugin. however

the text of the external link in the` $:/plugins/OokTech/Bookmarks/Bookmarks` List becomes unreadable if you have a dark/grey tiddler background and i cant find the place to change the colour.

can you tell me what to change in the pallet or is it hard-coded somewhere?

![billede](https://user-images.githubusercontent.com/2864323/48410446-e4f43b80-e73e-11e8-8251-36ac492994b4.png)

thanks

**Comment by inmysocks** [comment]
The plugin doesn't set any colours so it should be using whatever the default it. I am not sure where the external link colours are set or how to change them.

**Comment by hchaase** [comment]
This is strange... 
I changed the external-link-visited colour in the colour pallet. other places in the wiki the colour for external visited links change but nothing happens in the bookmarklist. Maybe they are not external links but something else, but I cant figure out what. Ive tried changing many different colours in the colour pallet and nothing seems to affect them. I don't think there is anything in the pallet with that colour.

_i_n the display template they are in html `<a    href={{!!url}}    target="_blank">` tags, could the html overwrite something maybe?

**Comment by hchaase** [comment]
Ive tried to change every! colour in the colour pallet, but neither the visited nor the non visited link colour change in the bookmark list. so the colour pallet have no control over them.

btw. do questions like this belong on github or in the tw forum?

**Comment by hchaase** [comment]
I solved the mystery.

apparently for the the colour pallet to effect external links they need the `class="tc-tiddlylink-external"`

your plug in **and** the markdown phraser don't have that. I have changed the display template. And note the deleted line break (it caused a hyperlinked space instead of just a space, after the link)

```
<a
    href={{!!url}}
    target="_blank" class="tc-tiddlylink-external"
>
    <$view
        field=title
    /></a>
 - {{!!note}} (<$link
    to={{!!title}}
>
    Open Tiddler
</$link>)
<br>
 
```

![billede](https://user-images.githubusercontent.com/2864323/48480559-2c47fe00-e80b-11e8-823d-5ade05533e42.png)
